charts/cinder-csi-plugin/values.yaml

```diff
@@ -160,6 +160,12 @@ secret:
 
 storageClass:
   enabled: true
+  parameters:
```

Comment:
is this sample ? so should be comment out?

Reply:
I don't really believe commenting our things in helm's values but always like to put each & every change behind a toggle to maintain backward compatibility. It's not a sample but actual values file, & I have kept all changed behind the `storageClass.parameters.Enabled` toggle, so this is a backward compatible change where folks if want can enable & by default it won't take effect.